### PR TITLE
Requested columns from data scientists DEVAS-1605

### PR DESCRIPTION
**Pull request by ayyazdaniaryan**
Added anonymizing users information for export. Added columns for post id, locale of content, original language column, etc.
TODO: Check to ensure Bigdatext service does not crash with these changes

### CODE REVIEW

File: assembl/static2/js/app/components/administration/exportSection.jsx [hunk 1]
```diff
@@ -79,8 +82,14 @@ class ExportSection extends React.Component<ExportSectionProps> {
         <div className="admin-content">
           {withLanguageOptions &&
             handleTranslationChange &&
-            handleExportLocaleChange && (
+            handleExportLocaleChange &&
+            handleAnonymousChange && (
               <FormGroup>
+                <Checkbox
+                  onChange={() => { handleAnonymousChange(); }}
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/administration/exportSection.jsx]: just use `onChange={handleAnonymousChange}` here

File: assembl/static2/js/app/components/administration/survey/step3.jsx [hunk 2]
```diff
@@ -38,17 +40,24 @@ class Step3 extends React.Component<Props, State> {
     this.setState({ translate: shouldTranslate });
   };
 
+  handleAnonymousChange = () => {
+    this.setState({ isAnonymous: !this.state.isAnonymous });
```
Comment (vincentfretin) [inline assembl/static2/js/app/components/administration/survey/step3.jsx]: You souldn't access this.state like that. You should use this syntax: `this.setState((prevState) => ({isAnonymous: !prevState.isAnonymous}))`

File: assembl/static2/js/app/pages/threadAdmin.jsx [hunk 3]
```diff
@@ -32,12 +33,18 @@ class ThreadAdmin extends React.Component {
     this.setState({ translate: shouldTranslate });
   };
 
+  handleAnonymousChange = () => {
+    this.setState({ isAnonymous: !this.state.isAnonymous });
```
Comment (vincentfretin) [inline assembl/static2/js/app/pages/threadAdmin.jsx]: here too

File: assembl/static2/js/app/utils/translations.js [hunk 4]
```diff
@@ -680,7 +680,8 @@ const Translations = {
         title: "Exporter les données du débat",
         defaultSectionTitle: "Exporter les données",
         taxonomySectionTitle: "Exporter les taxonomies",
-        translateTheMessagesIn: "Traduire l'ensemble des messages en:"
+        translateTheMessagesIn: "Traduire l'ensemble des messages en:",
```
Comment (vincentfretin) [inline assembl/static2/js/app/utils/translations.js]: add a space before `:` in French

File: assembl/static2/js/app/components/administration/survey/step3.jsx (hunk 2 above)
Comment (cedricmessiant) [reply assembl/static2/js/app/components/administration/survey/step3.jsx]: More info here: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous

Comment (ayyazdaniaryan) [reply assembl/static2/js/app/components/administration/survey/step3.jsx]: Thanks for the heads up. Changes made.

**Comment by ayyazdaniaryan** [comment]
Looks fine for verbatims:
https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html#pandas-read-csv
`usecols` field is used.

**Pull request merged**